JWBestPracticeApps/JWFairPlayDrm/JWFairPlayDrmViewController.swift

```diff
@@ -57,7 +63,11 @@ class JWFairPlayDrmViewController: JWBasicVideoViewController, JWDrmDataSource,
             let configuration = URLSessionConfiguration.default
             let session = URLSession.init(configuration: configuration, delegate: self, delegateQueue: nil)
             let postDataTask = session.dataTask(with: request as URLRequest, completionHandler: { (data, response, error) in
-                completion(data, nil, nil)
+                guard let data = data else {
+                    session.invalidateAndCancel()
+                    return
+                }
+                completion(data, Date.distantFuture, "application/octet-stream")
```

Comment:
The Xcode throws me these errors:
> 'nil' is not compatible with expected argument type 'Date'.
> 'nil' is not compatible with expected argument type 'String'

So I think it is because the NS_ASSUME_NONNULL_BEGIN was added to the JWDrmDataSource protocol and I can't return nil through the block.

Anyways I remember that I tested this target and still working well.

Reply:
ah that's a bug in the SDK, those can be null. I'll make a ticket